### DISCUSSION TITLE
Keypad examples and or more config examples

### DISCUSSION THREAD

**Discussion by rodal2** [meow-edit/meow#516] [General]
Hello, I might be misunderstanding how the keypad is supposed to be used and I think it would be a good idea to have more example meow-configurations files linked somewhere on the github. More examples is always good, I also like the idea of having some user-contributed configurations somewhere easily findable.

What I am currently trying to do (just for experimentation sake) is to bind "C-c u" as the save-buffer function but only in org-mode to then be able to access it via the keypad as "SPC u". My current attemp is doing:

(with-eval-after-load "org"
  (define-key org-mode-map (kbd "C-c u") #'save-buffer))
(meow-setup)

Using this I can not use "SPC u" to save an org-mode-buffer, which is the behaviour I expected. Am I wrong about this or is some other part of my configuration getting on the way?

Thank you for your time and the amazing software

**Comment by DogLooksGood** [comment]
Hi, I guess you need this:
```
(setq meow-keypad-leader-dispatch "C-c")
```

[The docs about leader](https://github.com/meow-edit/meow/blob/master/TUTORIAL.org#the-leader-keymap)

**Reply by rodal2** [reply]
That is exactly what I needed. Thank you very very much :).
